Source/WebKit/UIProcess/ios/forms/WKDateTimeInputControl.mm

```diff
@@ -309,6 +379,9 @@ - (instancetype)initWithView:(WKContentView *)view
     case WebKit::InputType::Month:
         mode = yearAndMonthDatePickerMode;
         break;
+    case WebKit::InputType::Week:
+        self = [super initWithView:view control:adoptNS([[WKDateTimePicker alloc] initWithView:view])];
```

Comment:
Echoing my comment from above, I don't think we need a separate initializer here.